generator/PythonGenerator.py

```diff
@@ -0,0 +1,622 @@
+"""
+Generates openMINDS classes.
+
+"""
+
+import glob
+import json
+import os
+import re
+from typing import List
+from collections import defaultdict
+import warnings
+
+from generator.commons import (
+    JinjaGenerator,
+    TEMPLATE_PROPERTY_TYPE,
+    TEMPLATE_PROPERTY_LINKED_TYPES,
+    SchemaStructure,
+    TEMPLATE_PROPERTY_EMBEDDED_TYPES,
+    SCHEMA_FILE_ENDING,
+    ROOT_PATH,
+    EXPANDED_DIR,
+    find_resource_directories)
+
+LIST_CLASSES_TEMPATE = '''
+
+def list_kg_classes():
+    """List all KG classes defined in this module"""
+    return [obj for name, obj in inspect.getmembers(sys.modules[__name__])
+           if inspect.isclass(obj) and issubclass(obj, KGObject) and obj.__module__.startswith(__name__)]
+'''
+
+# for backwards compatibility or to increase clarity we remap certain
+# names from the schemas when creating Python attribute names
+name_map = {
```

Comment:
fairgraph renames certain properties for backwards compatibility, but I don't think openMINDS-Python should do this.

(and probably this will not be done in fairgraph >= 1.0)

Reply:
I also discussed this @lzehl and she agrees.